### PR TITLE
Added more details for FAQs.

### LINKED ISSUE #95: ReadMe: Add FAQ Section
**I was looking forward to add FAQs Section in the ReadMe file.**

### PR DESCRIPTION
**Pull request by rk05-lenavo**
Fixed #95

## Related Issue 

- Info about the related issue 

Closes: #[issue number that will be closed through this PR]

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

**Comment by rk05-lenavo** [comment]
@praveenscience Sir I have Created the Pull Request. Can You Please review it.

File: ReadMe.md [hunk 1]
```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 
 ## FAQs
 
+#### About OpenSource Projects
+
+ OpenSource Projects means anybody is free to use, study and modify your project for any purpose. The open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love. means anybody is free to use, study, modify, and distribute your project for any purpose.
```
Comment (praveenscience) [inline ReadMe.md:171]: Looks good... Please do a grammar check and make it right.

File: ReadMe.md [hunk 2]
```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 
 ## FAQs
 
+#### About OpenSource Projects
```
Comment (praveenscience) [inline ReadMe.md:169]: Make it in a question way, and also, put this as a Level 3 heading. If not, add another Level 3 heading above it, saying General FAQs.

File: ReadMe.md [hunk 3]
```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+* $ git diff      (This will show you the exact changes in green color)
+* $ git add . (This will add all modifed files)
+* $ git commit -m "Fixed ISSUE
+   >
+   > ISSUE NO."
+* $ git push origin <Current branch name>.
+* Copy the Pull request link.
+* Open the link in your browser.
+* Scroll down to see all  changes you have done.
+* Click on option Create Pull Request.
```
Comment (praveenscience) [inline ReadMe.md:195]: Already Available. You can remove this. 😊

File: ReadMe.md [hunk 4]
```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+
+
+### Never mess with your master branch
+
+The master branch is deployable. It is your production code. The master branch is meant to be stable, and it is the social contract of open source software to never, ever push anything to master that is not tested, or that breaks the build.  If you make changes to the master branch of a group project while other people are also working on it, your on-the-fly changes will ripple out to affect everyone else and very quickly there will be merge conflicts.
```
Comment (praveenscience) [inline ReadMe.md:200]: Super cool. 😊  Again, grammar changes please?

File: ReadMe.md [hunk 5]
```diff
@@ -193,7 +227,7 @@ As it's an Agile Methodology, we're hosting weekly Live Streams and assigning th
 
 #### Can I work on an already opened issue?
 
-You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue.
+You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue (in which you have to create new issue by reading about that Project,searchig the bug present in it).
```
Comment (praveenscience) [inline ReadMe.md:230]: Grammar changes, spelling mistakes to be corrected please. 😊

**Review by praveenscience — CHANGES_REQUESTED** [review]
Get rid of all the non needed files inside `.idea` folder and please address the changes. 😊  Great attempt, thanks.

File: ReadMe.md (hunk 2 above)
Comment (rk05-lenavo) [reply ReadMe.md:169]: Okay sir

File: ReadMe.md (hunk 1 above)
Comment (rk05-lenavo) [reply ReadMe.md:171]: Okay sir

File: ReadMe.md (hunk 3 above)
Comment (rk05-lenavo) [reply ReadMe.md:195]: Okay sir.

File: ReadMe.md (hunk 4 above)
Comment (rk05-lenavo) [reply ReadMe.md:200]: Sure sir

File: ReadMe.md (hunk 5 above)
Comment (rk05-lenavo) [reply ReadMe.md:230]: sure sir.

**Comment by rk05-lenavo** [comment]
@praveenscience Sir can you now check. If it seems okay to you Please merge my Pull request.

File: ReadMe.md [hunk 6]
```diff
@@ -164,7 +164,17 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Salma Shaik
 * Prachi Mane
 
-## FAQs
+## General FAQs.
+
+#### What is OpenSource Projects
+
+ OpenSource Projects means anybody is free to use, study and modify your project. The open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love.
```
Comment (praveenscience) [inline ReadMe.md:171]: - Project, not projects.
- an Open Source.
- remove the space before the first line.
- remove the space after the para.

File: ReadMe.md [hunk 7]
```diff
@@ -164,7 +164,17 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
 
-## FAQs
+## General FAQs.
+
+#### What is OpenSource Projects
+
+ OpenSource Projects means anybody is free to use, study and modify your project. The open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love. 
+
+
+### Why is the master so important to not mess with?
```
Comment (praveenscience) [inline ReadMe.md:174]: change master to `master` branch. (add back ticks and branch)

File: ReadMe.md [hunk 8]
```diff
@@ -164,7 +164,17 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+
+The master branch is deployable. It is your production code, ready to roll out into the world. The master branch is meant to be stable, and it is the social contract of open source software to never, ever push anything to master that is not tested. If you make changes to the master branch while other people are also working on it, your on-the-fly changes will ripple out to affect everyone else and very quickly there will be merge conflicts. So,branching is a technique that gives a developer or group of developers the opportunity to create a new copy of a project without affecting the original which is the master branch. This allows developers to experiment with a branch and if the experiment is successful, Git makes it easy to incorporate the experimental elements into the Master.
```
Comment (praveenscience) [inline ReadMe.md:176]: - space after comma.
- comma after "group of developers"
- comma before which.

File: ReadMe.md [hunk 9]
```diff
@@ -193,7 +203,7 @@ As it's an Agile Methodology, we're hosting weekly Live Streams and assigning th
 
 #### Can I work on an already opened issue?
 
-You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue.
+You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue (means by creating new issue).
```
Comment (praveenscience) [inline ReadMe.md:206]:
```suggestion
You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue (either by creating a new issue or requesting you to create a new issue).
```

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please, can you fix these?

**Comment by rk05-lenavo** [comment]
@praveenscience Sir Please check it once.

**Comment by praveenscience** [comment]
Awesome, content is great, but can you please get rid of the `.idea` folder? Why is that even coming in? 🤔

**Review by praveenscience — CHANGES_REQUESTED** [review]
Awesome, content is great, but can you please get rid of the `.idea` folder? Why is that even coming in? 🤔

**Comment by rk05-lenavo** [comment]
@praveenscience Sir is it now Okay I've deleted all .idea files. Actually  myself don't Know how it came. Maybe because I had used the command $ git add .   so, but I'm not sure about it. Sir is it now okay?

**Comment by praveenscience** [comment]
Lemme check.

File: ReadMe.md [hunk 10]
```diff
@@ -164,7 +164,16 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Salma Shaik
 * Prachi Mane
 
-## FAQs
+## General FAQs.
```
Comment (praveenscience) [inline ReadMe.md:167]:
```suggestion
## FAQs

### General FAQs
```

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by rk05-lenavo** [comment]
@praveenscience Sir now is it okay?

**Review by praveenscience — APPROVED** [review]

File: ReadMe.md [hunk 11]
```diff
@@ -164,7 +164,16 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Salma Shaik
 * Prachi Mane
 
-## FAQs
+## General FAQs
```
Comment (praveenscience) [inline ReadMe.md:167]: Missing Section heading.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md (hunk 11 above)
Comment (rk05-lenavo) [reply ReadMe.md:167]: Sir I didn't get that. How to do that.

Comment (praveenscience) [reply ReadMe.md:167]: I'll fix it...

Comment (praveenscience) [inline ReadMe.md:167]:
```suggestion
## FAQs

### General FAQs
```

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md (hunk 11 above)
Comment (rk05-lenavo) [reply ReadMe.md:167]: Okay Sir. Sir now can you merge my Pull rerquest. If it seems okay to you.

File: ReadMe.md [hunk 12]
```diff
@@ -166,6 +166,17 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+
+### Why is the `master` branch so important to not mess with?
+
+The master branch is deployable. It is your production code, ready to roll out into the world. The master branch is meant to be stable, and it is the social contract of open source software to never, ever push anything to master that is not tested. If you make changes to the master branch while other people are also working on it, your on-the-fly changes will ripple out to affect everyone else and very quickly there will be merge conflicts. So, branching is a technique that gives a developer, group of developers the opportunity to create a new copy of a project without affecting the original, which is the master branch. This allows developers to experiment with a branch and if the experiment is successful, Git makes it easy to incorporate the experimental elements into the Master.
+
+
```
Comment (praveenscience) [inline ReadMe.md:179]:
```suggestion

```

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md [hunk 13]
```diff
@@ -166,6 +166,15 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 
 ## FAQs
 
+### General FAQs
+
+#### What is OpenSource Project
```
Comment (praveenscience) [inline ReadMe.md:171]:
```suggestion
#### What is an Open Source Project?
```

File: ReadMe.md [hunk 14]
```diff
@@ -166,6 +166,15 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+
+OpenSource Project means anybody is free to use, study and modify your Project. An Open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various Project, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love.
```
Comment (praveenscience) [inline ReadMe.md:173]:
```suggestion
Open Source Project denotes the project's source code is free for anyone to use, study and modify. An Open Source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love.
```

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md [hunk 15]
```diff
@@ -166,6 +166,15 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+
+### Why is the `master` branch so important to not mess with?
+
+The master branch is deployable. It is your production code, ready to roll out into the world. The master branch is meant to be stable, and it is the social contract of open source software to never, ever push anything to master that is not tested. If you make changes to the master branch while other people are also working on it, your on-the-fly changes will ripple out to affect everyone else and very quickly there will be merge conflicts. So, branching is a technique that gives a developer, group of developers the opportunity to create a new copy of a project without affecting the original, which is the master branch. This allows developers to experiment with a branch and if the experiment is successful, Git makes it easy to incorporate the experimental elements into the Master.
```
Comment (praveenscience) [inline ReadMe.md:177]:
```suggestion
The master branch is deployable. It is your production code, ready to roll out into the world. The master branch is meant to be stable, and it is the social contract of open source software to never, ever push anything to master that is not tested. If you make changes to the master branch while other people are also working on it, your on-the-fly changes will ripple out to affect everyone else and very quickly there will be merge conflicts. So, branching is a technique that gives a developer, group of developers the opportunity to create a new copy of a project without affecting the original, which is the master branch. This allows developers to experiment with a branch and if the experiment is successful, Git makes it easy to incorporate the experimental elements into the Master.

```

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
@rk05-lenavo Merged, thanks. 😊

**Comment by rk05-lenavo** [comment]
@praveenscience Thank You so much sir.

**Pull request merged**